connectors/src/connectors/slack/index.ts

```diff
@@ -235,16 +235,23 @@ export async function cleanupSlackConnector(
         return new Err(new Error("SLACK_CLIENT_SECRET is not defined"));
       }
 
-      const slackClient = await getSlackClient(connector.id);
-      const deleteRes = await slackClient.apps.uninstall({
-        client_id: SLACK_CLIENT_ID,
-        client_secret: SLACK_CLIENT_SECRET,
-      });
-      if (deleteRes && !deleteRes.ok) {
-        return new Err(
-          new Error(
-            `Could not uninstall the Slack app from the user's workspace. Error: ${deleteRes.error}`
-          )
+      try {
+        const slackClient = await getSlackClient(connector.id);
+        const deleteRes = await slackClient.apps.uninstall({
+          client_id: SLACK_CLIENT_ID,
+          client_secret: SLACK_CLIENT_SECRET,
+        });
+        if (deleteRes && !deleteRes.ok) {
+          return new Err(
+            new Error(
+              `Could not uninstall the Slack app from the user's workspace. Error: ${deleteRes.error}`
+            )
+          );
+        }
+      } catch (e) {
+        logger.error(
+          { connectorId: connectorId, error: e },
+          "Could not uninstall the Slack app from the user's workspace."
         );
```

Comment:
Should we return new Err as well?

Reply:
No, the goal of this fix was to be able to delete a connector even when the access token is revoked. So we log it for auditing, but we keep going.

Reply:
But then why do you return a new Err just above?

Reply:
As per IRL we should error here. This silent error will create more problems than it solve. It's better to have an explicit error to the user (us) than silently fail and create API errors on each webhook